eng/Version.Details.xml

```diff
@@ -1,83 +1,83 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="8.0.100-rc.2.23472.8" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
+    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="8.0.100-rtm.23505.1" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
       <Uri>https://github.com/dotnet/installer</Uri>
-      <Sha>e13fd5c19e383f99132b1bfd2496857db089b680</Sha>
+      <Sha>77a7628585b8f52970455233344ef3a9ce7c9bcb</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="8.0.0-rc.2.23457.7">
+    <Dependency Name="Microsoft.NETCore.App.Ref" Version="8.0.0-rtm.23506.12">
```

Comment:
This is a newer runtime than ours:

https://github.com/xamarin/xamarin-android/blob/release/8.0.1xx/eng/Version.Details.xml

It's probably ok, if it works.